packages/neutrino/package.json

```diff
@@ -2,16 +2,11 @@
   "name": "neutrino",
   "version": "8.2.0",
   "description": "Create and build JS applications with managed configurations",
-  "main": "src/index.js",
-  "bin": {
-    "neutrino": "./bin/neutrino.js"
```

Comment:
Do you think we should leave a bin script that outputs something like "The neutrino command has been removed. See migration guide"?

Reply:
Works for me.